### DISCUSSION TITLE
Administrator

### DISCUSSION THREAD

**Discussion by Justman100** [forem/forem#20299] [Q&A]
Since the official forum is only contaminated with advertisements, I would like to apply for administrator permissions

**Comment by Justman100** [comment]


**Comment by Justman100** [comment]
@lightalloy @maestromac @benhalpern

Since the trusted permissions are not enough, I still ask for administrator permissions! Since the forum is apparently no longer or hardly maintained by the maintainers, I can be granted the authorisations in trust

It's really important! 3.75/4 of all topics in the forum are just adverts and spam

**Comment by Justman100** [comment]
@maestromac

**Comment by TERNOX** [comment]
This is a really good idea, now I have the full impression that the site is dead, just like forem in general.

**Reply by Justman100** [reply]
🤷

**Comment by maestromac** [comment]
Can you please clarify which "forum" are you referring to?

**Reply by Justman100** [reply]
[The official Forum](https://forem.dev)

**Reply by Justman100** [reply]
@maestromac

There?

**Reply by maestromac** [reply]
We value your willingness to contribute as a moderator. However, at this moment, our resources are limited, and we can not accommodate additional moderation responsibilities without proper oversight. We appreciate your understanding.

**Reply by Justman100** [reply]
Dude, 3.75/4 of the entire forum consists only damn advertisements! And you're talking about *we have no resources*? It's not even damn 20 seconds to give me this permissions!

I don't really care, but if I don't get the permissions, the forum could shut down directly

**Comment by Justman100** [comment]
Well, since one is apparently too incompetent to realise that the forum is contaminated with advertising and doesn't even take 20 seconds to simply give me these permissions, Forem is dead for me! Because simply no pig is able to do anything! What dirty asseholes!

**Comment by Justman100** [comment]
Then Forem will simply continue to die! I shouldn't really care

**Reply by TERNOX** [reply]
It's looks like new Forem-forum will be there https://forem.com/

**Reply by Justman100** [reply]
> It's looks like new Forem-forum will be there https://forem.com/

You mean [dev.to](https://dev.to)?
